frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -487,14 +495,25 @@ const Dashboard: FunctionComponent<DashboardParams> = ({
 
             <TitleText>your progress</TitleText>
             <ProgressStatsContainer>
-              <Stat number={14} description={"day streak"} />
-              <Stat number={25} description={"words learned"} />
               <Stat
-                number={4}
+                number={dayStreak(
+                  currentUser.daysActive.map((val) => new Date(val))
+                )}
+                description={"day streak"}
+              />
+              <Stat
+                number={totalWordsLearned}
+                description={"words learned"}
+              />
+              <Stat
+                number={currentUser.sessionId + 1}
                 description={"assessments completed"}
               />
               <Stat
-                number={3}
+                number={
+                  currentUser.sessionId +
+                  (currentUser.onAssessment ? 1 : 0)
```

Comment:
could you explain this? I'm not quite getting it, maybe I'm confused about what sessionId is.

Reply:
So sessionId is the current session. -1 stands for pre-assessment, and then goes through 9. You have always done sessionId + 1 assessments since you had to do 1 assessment per session before the current one. You have done sessionId interventions, since we don't have one for -1. If you are on an assessment, then you did the intervention for the current session, so you have done sessionId + 1 interventions. Does that make sense?

Reply:
Got it, makes sense!